ansible/group_vars/all/defaults.yaml

```diff
@@ -31,7 +31,7 @@ critools_deb: "{{ crictl_version }}-00"
 critools_rpm: "{{ crictl_version }}-0"
 
 # Containerd container runtime version.
-containerd_version: "1.6.10"
+containerd_version: "1.6.15"
```

Comment:
Any reason in particular we need to bump this as a part of this PR?

Reply:
Thanks for bringing this up.
This is the latest and only containerd `1.6.x` version we are building for rocky. https://teamcity.mesosphere.io/buildConfiguration/MesosphereOnly_ClosedSource_DkpReleases_UpstreamBuilds_Containerd_Rocky91?branch=&buildTypeTab=overview&mode=builds

I agree this will impact other OS builds too. Do you see bumping patch version of containerd here would require any additional testing and considerations?

I can manually build containerd `1.6.10` for rocky too and revert this change.

Reply:
I think it should be fine to do as a part of this PR. I was just wondering why you did it.

Reply:
I think we do want to bump the version but I would really prefer it as a separate PR, and then base this PR on that. WDYT?

Reply:
Thats fair. I will revert this commit and file separate PR

Reply:
reverted back to containerd 1.6.10